app/views/trainees/start_dates/edit.html.erb

```diff
@@ -4,9 +4,15 @@
   <%= render DynamicBackLink::View.new(@trainee, text: t(:back)) %>
 <% end %>
 
-<%= register_form_with model: @trainee_start_date_form, url: trainee_start_date_path, local: true do |f| %>
-  <%= f.govuk_date_field :commencement_date, legend: { text: t("views.forms.training_details.commencement_date.label"), size: "l", tag: "h1" } %>
-  <%= f.govuk_submit "Continue" %>
-<% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= register_form_with model: @trainee_start_date_form, url: trainee_start_date_path, local: true do |f| %>
+      <%= f.govuk_date_field :commencement_date, legend: {
+        text: t("views.forms.start_dates.commencement_date.label"), size: "l", tag: "h1"
```

Comment:
Is this size and `h1` correct? I don't see this output in the rendered markup - but we don't want an h1 on the date field.